EMBEDS_README.md

```diff
@@ -0,0 +1,21 @@
+# Let's get to know each other a little better
+Welcome (temporarily) to the team!  Although this engagement isn't permanent, let's take this opportunity to set a few expectations.  Many of the tenants from my [Manager README for full-time direct reports](https://github.com/esquireofoz/Manager-README/blob/master/README.md) apply, but the nature of an embedding arrangement have some additional nuances we'll cover here.
+
+---
+## What I expect of you:
+* Before you spend any time with the team, you will recieve an embedding template where we will set goals, timeline and the scope of the engagement.
+
+* You will not be required to participate in anything related to on-call (unless you wish to learn about it)
```

Comment:
Missing period (though I get these are list items so no need for one - in that case, may as well remove the others, just picking a side for consistency sounds good to me) 👍  Also, hurray for no oncall! 😅

Reply:
One thing that's interesting - if we're added to the team (ex. I'm added to @tumblr/sredatabase) are we doing pull request reviews where the team's tagged? I think it's still good to take a look, but since we're not familiar with the domain, should keep getting the rest of the team's eyes as well 😁

Reply:
Also a small note just for reassurance may be helpful, that we understand you might still have duties from your original team like PR reviews (I'm still in the android and php review groups 😅 ) and RFC meetings. I think those are understandable and up to the person to juggle the time commitment, but we'll try to clear the deck of any project work and tickets from the original team.

Reply:
Great suggestion about code reviews on requests for the team where the embedding is occurring!  I'm going to keep this README generic, but specifics regarding these implementation details will be added to the internal onboarding documentation which is presented at the start of an embedding.

Also, good point about reassurances, I'll add a little more color to the README.  Ultimately embeddings are what the people doing the embedding make of them, but the scope of the embedding work should be structured to maximize impact and interest.